components/theme/Button.tsx

```diff
@@ -13,6 +13,21 @@ type ButtonProps = {
   size?: 'lg' | 'sm'
   onClick?: Function
   outline?: boolean
+  testId?: string
```

Comment:
You dont have to add all the html attributes yourself to the type definition
Try this:
```
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  border?: boolean
  type?: 'success' | 'danger' | 'primary' | 'light' | 'info' | 'mute'
  color?: ColorTypes
  m?: '1'
  ml?: '2'
  size?: 'lg' | 'sm'
  outline?: boolean
}
```

One other concern I have is with the `type` prop, since thats also an html attribute.

Reply:
> One other concern I have is with the `type` prop, since thats also an html attribute, but is defined in the props as ` type?: 'success' | 'danger' | 'primary' | 'light' | 'info' | 'mute'`

I think this a bootstrap thing. It might be a good idea to change the variables to something like `btnType`?
This variable just affects what strings get added to the className, it's not actually passed into the `button` element.

Reply:
Yeah thats a good idea.